cmd/agent/check.go

```diff
@@ -139,17 +138,23 @@ func clearOldProcsFiles(baseDir string) {
 			continue
 		}
 
+		// get path type and values
 		baseName := path.Base(f.Name())
-		var id string
-		if seps := strings.Split(baseName, "_"); len(seps) != 2 {
+		tp, v, ptp := getPathType(baseName)
+		if ptp != idPathType {
 			continue
-		} else {
-			id = seps[1]
 		}
 
-		if _, ok := procs[id]; !ok {
+		// path format is type_id
+		if _, ok := procs[v]; !ok {
 			// we need remove unnecessary files
 			os.RemoveAll(path.Join(baseDir, baseName))
+
+			// if log type, then we just rename dir
+			if tp == logType {
+				newName := fmt.Sprintf("%s_%s_%d", tp, v, time.Now().Nanosecond())
+				os.Rename(path.Join(baseDir, baseName), path.Join(baseDir, newName))
```

Comment:
that is ok, because we only call rename func, but no checking rename error.
so if old path does not exist, i think rename only return an error, but in fact we ignore it.